debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/logwriter/CommitLogWriterFlushStrategy.java

```diff
@@ -89,11 +93,19 @@ public String getHost() {
     @Override
     public void flush(Scn currentScn) {
         try {
+            if (!Strings.isNullOrEmpty(databasePdbName)) {
```

Comment:
Absolutely.  I've wanted to see if we could encapsulate the entire concept of PDB vs non-PDB inside of the connection class so that external callers don't need to be concerned with that bit at all.  

This would have the benefit to simplifying the test suite when running against a PDB and non-PDB environment as well as enforcing that runtime queries should really define method on `OracleConnection` and we don't have SQL fragments lying around in various parts of the other connector classes.

Logged https://issues.redhat.com/browse/DBZ-6452